plugins/remember.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import sys
+
+if len(sys.argv) < 3:
+    reply = "Missing parameters"
+else:
+    args = sys.argv
+    args.pop(0)
+    args.pop(0)
+
+    reply = "*{} will remember that*".format(" ".join(args))
+
+try:
+    print(reply)
+except NameError:
+    pass
```

Comment:
while the above works it looks so much worse than just starting with 

```
reply = ""
```

at the top of the file and then not doing all that extra catching